docs/user-guide/multicontract/index.md

```diff
@@ -0,0 +1,675 @@
+
+    env e; uint256 amount;
+    deposit(e, amount);
+
+    mathint balance_after = assetBalance();
+
+    assert balance_after == balance_before + amount,
+        "deposit must increase the underlying balance of the pool";
+}
+```
+
+This rule makes a call to `Pool.deposit(...)`, which in turn makes a call to
+`asset.transferFrom(...)`; to understand the behavior of `deposit` the Prover
+must also reason about the `Asset` contract.  If we verify the rule without
+giving the Prover access to the `Asset` code, the call to `transferFrom(...)`
+will be unresolved.
+
+By default, the Prover will handle calls to unresolved functions by assuming
+they can do almost anything &mdash; we say that the Prover "{term}`havocs <havoc>`"
+some part of the state.  The part of the state that is havoced depends on the
+type of call: calls to view functions are allowed to return any value but can
+not affect storage (a `NONDET` summary), while calls to non-view functions are
+allowed to change the storage of all contracts in the system *besides the
+calling contract*[^reentrancy] (a `HAVOC_ECF` summary).  See
+{ref}`auto-summary` in the reference manual for complete details.
+
+[^reentrancy]: The Prover assumes that the external calls do not modify the
+  storage of the calling contract.  This assumption comes from an assumption
+  that the called code is non-reentrant.  If you are concerned about violations
+  caused by reentrancy, you can override this assumption using a `HAVOC_ALL`
+  summary; see {ref}`havoc-summary` for details.
+
+We can see this behavior by verifying the `integrityOfDeposit` rule against the
+`Pool` contract without giving the Prover access to the `Asset` contract ([full script][havoc-script]):
+
+```sh
+$ certoraRun contracts/Pool.sol --verify Pool:certora/specs/pool_havoc.spec ...
+```
+
+In this case, the `integrityOfDeposit` rule fails.  To understand why, we can
+unfold the call trace for the call to `deposit`:
+
+![Call trace for `integrityOfDeposit` with `deposit` method unfolded to show DEFAULT HAVOCs for calls to `balanceOf` and `transferFrom`](no-link-call-trace.png)
+
+Here we see that the calls to `transferFrom` and `balanceOf` are marked with
+"DEFAULT HAVOC".  This means that the Prover lets the call to
+`transferFrom` to change the balances any way it likes.  In fact, calls to
+`asset.balanceOf(...)` are also unresolved, so the Prover can choose any return
+value that causes a counterexample.  In this case, we can see that the Prover
+chose `3` for the first return value of `balanceOf` and `9`
+for the last return value of `balanceOf`:
+
+![Call trace for `integrityOfDeposit` on `Pool` showing call to `assetBalance` with internal havoced call to `balanceOf`, returning 3 in once place and 9 in another](no-link-variables.png)
+
+The "Call Resolution" tab on the report provides more information about all of
+the unresolved method calls within the contract and how they are resolved by
```

Comment:
would external calls be a better term? because you see both resolved and still unresolved.